components/Events.js

```diff
@@ -80,7 +80,7 @@ Event.propTypes = {
     venue: PropTypes.shape({
       name: PropTypes.string.isRequired,
       city: PropTypes.string.isRequired,
-    }).isRequired,
+    }),
```

Comment:
very timely fix :D

Reply:
Haha, took me a full day to get it. You mean because of virtual corona meetings? No venue needed indeed :D